css/style.css

```diff
@@ -0,0 +1,405 @@
+body {
+    font-family: 'Josefin Sans', sans-serif;
+    -webkit-font-smoothing: antialiased;
+    position: relative;
+    margin: 0px;
+}
+ /* Header  */
+ header {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-color: #dee2ff;
+    
+}
+
+.hamburger-menu {
+    align-self: flex-end;
+    margin-top: 10px;
+    margin-right: 20px;
+}
+
+.hamburger-menu img {
+    width:100%;
+    height: 20px;
+}
+
+ .logo {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 5px 8px;
+    height: 150px;
+}
+
+.logo-icon {
+    width: 30%;
+    align-self: flex-start;
+}
+
+.logo-icon img {
+    width: 90%;
+    height: 90%;
+}
+
+.logo-text {
+    width: 70%;
+    align-self: flex-end;
+    padding: 10px 10px;
+}
+
+.logo-text p {
+    font-size: 1.05em;
+    font-style: italic;
+    text-align: right;
+    color: #2b2d42;
+}
+
+/* Main First Section */
+
+main {
+    background-color: #feeafa;
+    width: 100%;
+}
+.navbar {
+    display: none;
+    width: 100%;
+    font-size: 1.25em;
+}
+
+.navbar ul {
+    display: flex;
+    flex-flow: nowrap;
+    justify-content: flex-end;
+    align-items: center;
+ }
+
+.navbar ul li {
+   list-style-type: none;
+   margin: 10px;
+   padding: 5px;
+
+}
+.navbar ul li a {
+   text-decoration: none;
+   color: #9d8189;
+   color: black;
+}
+
+.navbar ul li a:hover {
+   color: orangered;
+}
+
+/* Main Second Section */
+
+.hero {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.hero-text {
+    padding: 10px;
+}
+
+.hero-text h2{
+    font-size: 2em;
+    padding: 5px;
+}
+
+.hero-text p {
+    font-size: 1em;
+    padding: 5px;
+    text-align: justify;
+}
+
+.hero-image {
+    padding: 10px;
+}
+
+.hero-image img {
+    width: 100%;
+}    
+
+/* Main Third Section */
+.container {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    flex-flow: wrap;
```

Comment:
These alignment rules ensure that the images are always visible, but doesn't fully control how they are laid out. Sometimes, there is just one image by itself on the second line. 
To avoid this, make sure the images are small enough to fit on one row, or set `flex-direction: column` until they have enough room to be in a row.